src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordWithOcrToExistingCaseTest.java

```diff
@@ -72,6 +75,34 @@ void should_update_case_with_ocr() throws Exception {
         assertThat(address.get("country")).isEqualTo(ocrCountry);
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    @Disabled("Functionality not implemented yet")
+    void should_return_payments_error_when_updating_case_with_ocr_having_pending_payments() throws Exception {
```

Comment:
These tests are a bit tricky, because what they do isn't really classification-specific. Maybe we should extract them all into a separate test class, whose responsibility is just to check if the application handles pending payments correctly?

Maybe someone else has some thoughts?